agent/app/agent_capability_windows_test.go

```diff
@@ -260,3 +260,47 @@ func TestAppendGMSACapabilitiesFalse(t *testing.T) {
 
 	assert.Equal(t, len(expectedCapabilities), len(capabilities))
 }
+
+func TestAppendFSxWindowsFileServerCapabilities(t *testing.T) {
+	var inputCapabilities []*ecs.Attribute
+	var expectedCapabilities []*ecs.Attribute
+
+	expectedCapabilities = append(expectedCapabilities,
+		[]*ecs.Attribute{
+			{
+				Name: aws.String(attributePrefix + capabilityFSxWindowsFileServer),
+			},
+		}...)
+
+	agent := &ecsAgent{
+		cfg: &config.Config{
+			FSxWindowsFileServerCapable: true,
+		},
+	}
+
+	capabilities := agent.appendFSxWindowsFileServerCapabilities(inputCapabilities)
+
+	assert.Equal(t, len(expectedCapabilities), len(capabilities))
+	for i, expected := range expectedCapabilities {
```

Comment:
Shouldn't the test iterate over `capabilities`? 
Can you not make use of other methods in https://godoc.org/github.com/stretchr/testify/assert

Reply:
this test is identical to ```TestAppendGMSACapabilities```